### ISSUE TITLE
New snappy-driver-installer possibly malicious?

### ISSUE THREAD

**Issue by Spunkie** [bcurran3/ChocolateyPackages_CCR#5]
I pinged you on chocolatey.org about releasing version `R545` of `snappy-driver-installer` but between then and you pushing a new version they released versions `R1741` and `1.17.4.2 `. Other users are now saying that these new versions includes adware though. 

So digging into this, I found out that at some point in the last 6 months the original lead developer handed the site/software off to some unknown company and that company has decided to monetize this software while still advertising it as free and open source software. What worse is they have turned off access to the original SVN repo so you can't even see or verify what kind of adware they are putting in these new versions and they are still saying it's GPL-v3 while having no actual source code available. The automated chocolatey virus scanner also changed for the new version, it went from [0/58](https://www.virustotal.com/en/file/08d439a3a94095bf78c27f461d1da00e71892ac94e51667a597699a3534c4a39/analysis/) to [12/59](https://www.virustotal.com/en/file/6a072db13655236f25f884532bc7fd896ebeb238dc31379cb6941cb675a1ae89/analysis/).

Luckily I also found one of the developer helping during the transition did not like what the new company was doing and decided to maintain his own free and open source fork. Which he just called "snappy driver installer origin", he also explained more about the situation on [this about page](http://snappy-driver-installer.org/about.html).

So the old and potentially malicious **_`snappy driver installer`_** is from this site:
> http://www.sdi-tool.org/

While the new **_`snappy driver installer origin`_** can be found on this site: 
> http://snappy-driver-installer.org/

I'm not really sure what would be the best way to handle this on chocolatey. Should we just made a new chocolatey pkg called `snappy-driver-install-origin` and then unpublish just the new `1.17.4.2 ` version or maybe just deprecate the entire `snappy-driver-installer`? Or would you want to keep publishing the new forked versions under `snappy-driver-installer` since there is no version overlap?

**Comment by bcurran3** [comment]
Drama! :)

I've unpublished 1.17.4.2 and pushed up v539 which was the last version on SourceForge from the beginning of last month.

I don't use this. Thanks for all your detective work. I took a look at the new site and it's all a little bit weird (being that he's not a programmer etc.). I'll do some thinking about the whole thing and probably push up the Origin version as a separate package in the future. I'll probably test it in a VM and check it. Please let me know you're experiences with it as I see you have definite interest.

**Comment by Spunkie** [comment]
>I've unpublished 1.17.4.2 and pushed up v539 which was the last version on SourceForge from the beginning of last month.

Great, thanks for your quick action on this.
___
>I don't use this. Thanks for all your detective work. I took a look at the new site and it's all a little bit weird (being that he's not a programmer etc.).

Ya the whole thing seemed a bit weird to me so I messaged the fork maintainer yesterday. Apparently he is a programmer but has never worked on _lower_ level language like C++, which I guess means he is a webdev or something. He also said he plans on distancing himself from the original project in the coming months with a new logo, website, possibly moving off sourceforge, etc. So hopefully there will be less confusion in the future. Anyways it seems the project is just in maintenance mode until he can find an experienced c++ dev to help out. 

>Please let me know you're experiences with it as I see you have definite interest.

Ya I definitely have an interest since I intend to use this software to manage drivers on my personal computer and have recommended this software to a few friends. If you would prefer, I would be happy build/maintain the _**`snappy driver installer origin`**_ pkg. I should have some free time this weekend to slap an AU build together for it. Shouldn't be to hard since you have already done most of the work with your pkg,

**Comment by Spunkie** [comment]
@bcurran3 Sorry, I forgot to update this issue. I ended up publishing the [`snappy-driver-installer-origin`](https://chocolatey.org/packages/snappy-driver-installer-origin/) pkg but the version # format of the software was changed shortly after so I had to depreciate it. The new updated pkg is simply [`SDIO`](https://chocolatey.org/packages/sdio/).

Also I think I found a bug with the shortcuts when putting together this pkg. Only 1 of the shortcuts sets the `-WorkingDirectory` which causes the software to act differently depending on which shortcut you use to open it.

1. If you open the desktop shortcut
    - It causes generated config files and downloaded drivers to save into `\ProgramData\chocolatey\lib\snappy-driver-installer\tools\`
   - I'm not 100% sure on this but I think this directory gets wiped out when running `choco upgrade` which would cause you to lose config files and previously downloaded driver packs?

2. If you then open the startmenu shortcut
    - It will act as a fresh install since it can't find the previously mentioned config files. 
    - This shortcut causes the generated config files and downloaded drivers to save into `\Windows\System32\`
